.github/CONTRIBUTING.md

```diff
@@ -381,4 +383,9 @@ $ git pull --ff-only upstream master
 $ git checkout new_branch
 ```
 
+#### Oh no! I've made my changes to `master` branch and have pushed it to my repo! What should I do now?
+- [Rename your `master` branch](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch)
+- [Recreate a new `master` branch](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch), change the repo source to `CleverRaven/Cataclysm-DDA` (base branch should be `master`)
+- Finally, make the new `master` branch [the default branch](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch)
```

Comment:
`git reset upstream/master`
`git push --repo=origin --force`
(assuming upstream is cleverraven and origin is your fork)

Not sure why you want to add random git trivia to the contributing guide though.

Reply:
Thanks, I forgot how to do things 8)